evcouplings/couplings/protocol.py

```diff
@@ -871,17 +1177,20 @@ def _postprocess_inference(ecs, kwargs, model, outcfg, prefix, generate_line_plo
     # only since enrichment code cannot handle multiple segments)
     if generate_enrichment:
         ext_outcfg["enrichment_file"] = prefix + "_enrichment.csv"
-        ecs_enriched = pairs.enrichment(ecs, score=score)
-        ecs_enriched.to_csv(ext_outcfg["enrichment_file"], index=False)
-
-        # create corresponding enrichment pymol scripts
-        ext_outcfg["enrichment_pml_files"] = []
-        for sphere_view, pml_suffix in [
-            (True, "_enrichment_spheres.pml"), (False, "_enrichment_sausage.pml")
-        ]:
-            pml_file = prefix + pml_suffix
-            enrichment_pymol_script(ecs_enriched, pml_file, sphere_view=sphere_view)
-            ext_outcfg["enrichment_pml_files"].append(pml_file)
+
+        ## TODO: need to make segment_aware_ec_enrichment compatible with new
```

Comment:
There shouldn't be any changes to the section modified here
1. enrichment() now handles enrichment for multi-segment ECs generically
2. enrichment_pymol_script() should use the segment ID to assign enrichment info to different chains

Any more specific enrichment outputs for complexes (e.g. only within each segment, like create here) should be made in complex protocol, not in generic function here